spec/i18n_routable/i18n_routable_spec.rb

```diff
@@ -149,55 +153,55 @@ def resolve(path, http_verb=:get)
 
     it 'should understand the difference between display locales and backend locales' do
       I18n.locale = :gibb
-      new_post_url.should eql "http://www.example.com/gibberish/posts/the-new"
+      new_post_url.should eql "http://www.example.com/gibb/posts/the-new"
```

Comment:
With enforce_available_locales set to true, we can no longer specify `{:gibberish => :gibb}` in the locales array, because setting available_locales calls `to_sym` on each item in the array. I changed it to just `gibb` now. Is this functionality that we need to support, or can we just leave it as `gibb`? I'm not exactly sure what display locales and backend locales means, but it looks like some of these tests might be useless without the previous functionality?